fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/WorldRenderEvents.java

```diff
@@ -68,13 +69,26 @@ private WorldRenderEvents() { }
 	 * identified and rebuilt but before chunks are uploaded to GPU.
 	 *
 	 * <p>Use for setup of state that depends on view frustum.
+	 *
+	 * <b>Can NOT be used to render anything after MC 1.21.2 due to a glClear call, use {@link WorldRenderEvents#BEFORE_TERRAIN} instead.
```

Comment:
building fails for me because of a missing closing tag for the bold text here, works after adding it

Reply:
The docs don't use closing tags for `<p>`. That format is pretty conventional for javadocs, and I think it's a special case in the Checkstyle rules. (In valid HTML, the end tag `</b>` can't be omitted but `</p>` can in this case.)

Reply:
I got this issue also, adding `</b>` fixed this.